From be7d6239e4971122dce0e0106341938e848dcf65 Mon Sep 17 00:00:00 2001
From: "Paul m. p. Peny" <mail.peny@free.fr>
Date: Thu, 4 Jan 2024 06:40:43 +0100
Subject: [PATCH 1/1] support emsdk >3.1.51

since https://github.com/pygame-community/pygame-ce/pull/2596 static.c build was silently ignored (commented by setuptools) because PNG/JPEG not set anymore in Setup
---
 buildconfig/Setup.Emscripten.SDL2.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/buildconfig/Setup.Emscripten.SDL2.in b/buildconfig/Setup.Emscripten.SDL2.in
index 0bd60a30..2ad66a94 100644
--- a/buildconfig/Setup.Emscripten.SDL2.in
+++ b/buildconfig/Setup.Emscripten.SDL2.in
@@ -6,7 +6,8 @@
 #MIXER = -lSDL2_mixer
 #SCRAP =
 #FREETYPE = -lfreetype -lharfbuzz
-
+PNG = -lpng
+JPEG = -ljpeg
 DEBUG =
 
 # these can build alone and object files merged with ar
-- 
2.29.2.windows.2

